src/colorpickr.js

```diff
@@ -441,8 +466,15 @@ class ColorPickr extends React.Component {
                 opacityHigh={opacityHigh}
               />
             </XYControl>
-            <div {...theme('slider', 'colorModeSlider', `colorModeSlider${channel.toUpperCase()}`)}>
+            <div
+              {...theme(
+                'slider',
+                'colorModeSlider',
+                `colorModeSlider${channel.toUpperCase()}`
+              )}
+            >
               <input
+                disabled={readOnly}
```

Comment:
Instead of always passing a `disabled` or `readonly` attributes can you conditionally them instead?